core/arch/aarch64/instr_create.h

```diff
@@ -1850,5 +1850,631 @@
 #define INSTR_CREATE_bic_sve_pred(dc, Zd, Pg, Zd_, Zm, width) \
     instr_create_1dst_4src(dc, OP_bic, Zd, Pg, Zd_, Zm, width)
 
+/* -------- Advanced SIMD three different ------------------------------ */
+
+/**
+ * Creates a SADDL vector instruction.
+ * \param dc      The void * dcontext used to allocate memory for the instr_t.
+ * \param Rd      The output register.
+ * \param Rm      The first input register.
+ * \param Rn      The second input register.
+ * \param width   The vector element width. Use either OPND_CREATE_BYTE(),
```

Comment:
IIUC width here applies to the 2 input registers and the width of the output register (next highest width) is implied. I think this is worth calling out here. Same for W variants and co